src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
```

Comment:
might've missed it first time, there should be three @font-face block, with same font-family but different font-weights, for Avenir, Avenir-Book, Avenir-Heavy

Reply:
I hope, now is better :)

`@font-face {
  font-family: Avenir, sans-serif;
  font-weight: 300;
  src: url(./fonts/Avenir.ttc);
  src: url(./fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: Avenir, sans-serif;
  font-weight: 900;
  src: url(./fonts/Avenir-Heavy.ttf);
}`